<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RipartPlugin</class>
 <widget class="QDialog" name="RipartPlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connexion au service Ripart</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;connexion au service&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Connecter</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLogin">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>11</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Votre login</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPwd">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>37</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Votre mot de passe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFoo">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>98</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Foo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLogin">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>11</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPwd">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>37</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/plugins/RipartPlugin/icon.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/plugins/RipartPlugin/images/logo_IGN.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 
 <connections>
  <connection>
   <sender>RipartPlugin</sender>
   <signal>accepted()</signal>
   <receiver>btnConnect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
